docs/src/components/header/MainLayoutHeader.vue

```diff
@@ -195,28 +195,28 @@
         }"
         class="dense-header q-pl-lg q-pr-md justify-between items-stretch"
       >
-        <div class="row justify-center items-center cursor-pointer">
-          <router-link :to="{ name: 'landing' }" class="row items-center">
+        <div class="row justify-start items-center cursor-pointer">
+          <router-link :to="{ name: 'landing' }" class="q-mr-sm">
             <img
               :src="`https://cdn.quasar.dev/logo-v2/svg/logo${$q.screen.gt.sm ? '-horizontal' : ''}${dark? '-dark':''}.svg`"
               alt="Quasar Logo"
               height="36"
-              :width="$q.screen.gt.sm ? 236 : 48"
             >
           </router-link>
           <q-separator
+            v-if="showDenseSeparator"
             :color="dark? 'brand-primary':'black-12'"
-            class="q-ml-lg"
+            class="q-mx-md"
             vertical
           />
           <nav-dropdown-btn
```

Comment:
I think the idea was to directly remove the version dropdown when on smaller viewports, not sure why I didn't, I'm pretty sure I did my tests by removing it, I possibly then forgot to add the v-if for some reason
Ask Nick if the version with denser dropdown is better